ruby_event_store_rom_sql/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+Further changes can be tracked at [releases page](https://github.com/RailsEventStore/rails_event_store/releases).
+
+### 0.1.0 (03.04.2018)
```

Comment:
We'll bump version to be in match with `RES_VERSION` on first release:
- https://github.com/RailsEventStore/rails_event_store/releases/tag/v0.15.0
- https://blog.arkency.com/why-we-follow-rails-repo-structure-in-rails-event-store/